lib/ansible/modules/database/postgresql/postgresql_schema.py

```diff
@@ -7,114 +7,138 @@
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
 
-
 ANSIBLE_METADATA = {'metadata_version': '1.1',
                     'status': ['preview'],
                     'supported_by': 'community'}
 
-
-DOCUMENTATION = '''
+DOCUMENTATION = r'''
 ---
 module: postgresql_schema
-short_description: Add or remove PostgreSQL schema from a remote host
+short_description: Add or remove PostgreSQL schema
 description:
-   - Add or remove PostgreSQL schema from a remote host.
-version_added: "2.3"
+- Add or remove PostgreSQL schema.
+version_added: '2.3'
 options:
   name:
     description:
-      - Name of the schema to add or remove.
+    - Name of the schema to add or remove.
     required: true
+    type: str
   database:
     description:
-      - Name of the database to connect to.
+    - Name of the database to connect to and create the schema.
```

Comment:
Given this can add/remove do we want to say `update`?
```suggestion
    - Name of the database to connect to and update the schema.
```

Reply:
Not really, but thanks! We make not only create, also remove.
I changed it "create" -> "add or remove" like in the description